File: csv/kotlin/read/csv_read.kt
// ------------------------------------------------------------------
/*
	csv_read.kt

					Apr/01/2015
*/
// ------------------------------------------------------------------
import	java.io.File
import	java.util.HashMap
 
// ------------------------------------------------------------------
fun main(args: Array<String>)
{
	println("*** 開始 ***")

	var file_in = args[0]

	println (file_in)

	var str_in = File(file_in).readText ()

	var dict_aa = str_to_dict_proc (str_in,",")

	dict_display_proc (dict_aa)

	println("*** 終了 ***")
}

// ------------------------------------------------------------------

File: csv/vb/read/csv_read.vb
'------------------------------------------------------------------------
'	csv_read.vb
'
'						Apr/07/2015
'
'------------------------------------------------------------------------
Imports System.IO
Imports System.Collections
'------------------------------------------------------------------------
Module Module1

'------------------------------------------------------------------------
Sub Main()
	Dim clArgs() As String = Environment.GetCommandLineArgs()
	console.writeLine("*** 開始 ***")
	Dim file_text As String = clArgs(1)
	console.writeLine (file_text)

	Dim dict_aa as Hashtable = text_to_dict_proc (file_text,",")

	dict_display_proc (dict_aa)

	console.writeLine("*** 終了 ***")
End Sub

'------------------------------------------------------------------------
End Module

'------------------------------------------------------------------------
